pages/solutions.js

```diff
@@ -1,84 +1,81 @@
+/* eslint-disable sort-imports */
```

Comment:
Not sure why this is added? Can't the `ArrowRight20` be below the other import?

Reply:
could it be, it is just a personal taste, I will remove... no problem